src/components/views/elements/AccessibleButton.js

```diff
@@ -27,8 +27,8 @@ import React from 'react';
 export default function AccessibleButton(props) {
     const {element, onClick, children, ...restProps} = props;
     restProps.onClick = onClick;
-    restProps.onKeyDown = function(e) {
-        if (e.keyCode == 13 || e.keyCode == 32) return onClick();
+    restProps.onKeyUp = function(e) {
```

Comment:
Because I was listening for onKeyUp further up the tree and wondering why it wasn't being correctly captured by the AccessibleButton when the focus was there, and it was because it's on KeyDown, so since its generally more correct to use KeyUp than KeyDown, I changed this one. It also was not passing the event through to the onClick handler which was breaking things.